embedding-reports/display-reports-in-applications/web-application/native-blazor-report-viewer/customizing/toolbar-customization.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Customize the Toolbar
+page_title: Customizing Native Blazor Viewer's Toolbar
+description: How to Customize the Toolbar of the Native Blazor Report Viewer
+slug: native-blazor-report-viewer/toolbar-customization
+tags: custom,editor,widget,parameter,blazor,toolbar,menu
+published: True
+position: 4
+---
+
+# Customize the Toolbar of the Blazor Native Report Viewer
+
+This article explains how to customize the default set and order of tools displayed in the Blazor Native ReportViewer toolbar.
+
+The customization is done similar to the [Blazor Editor component](https://docs.telerik.com/blazor-ui/components/editor/built-in-tools).
+
+## Available Tools
+
+Here is the list of all the Tools that may be used in the toolbar in their default order of appearance as may be seen in the [Native Blazor Report Viewer Overview]({%slug telerikreporting/embedding-reports/display-reports-in-applications/web-application/native-blazor-report-viewer/overview%}):
+
+* `NavigateBackward`
+* `NavigateForward`
+* `StopRendering`
+* `Refresh`
+* `FirstPage`
+* `PreviousPage`
+* `PageNumber`
+* `NextPage`
+* `LastPage`
+* `ToggleViewMode`
+* `Export`
+* `Print`
+* `SendEmail`
+* `ToggleDocumentMap`
+* `ToggleParametersArea`
+* `ZoomIn`
+* `ZoomOut`
+* `ToggleScaleMode`
+* `Search`
+
+The predefined set of all availabe tools is accessible from the `Telerik.ReportViewer.BlazorNative.Tools.ToolSets.Default` static property. It will return the List of the tools displayed by default in the ReportViewer.
+
+## Implementing the Customized Toolbar
+
+Use the `Tools` [option of the viewer]({%slug telerikreporting/embedding-reports/display-reports-in-applications/web-application/native-blazor-report-viewer/api-reference/options%}#options) to pass a custom ordered list of Tools for the Toolabar. for example, the below code removes the _Search_, _SendEmail_ and _zoom_ functionalities from the veewer and moves the _Refresh_ in the beginning of the toolabar:
+
+````CSharp
```

Comment:
I have seen the Blazor team use CSHTML instead of CSharp, it will probably highlight the code better